packages/regl-worldview/src/commands/Points.js

```diff
@@ -34,6 +34,8 @@ const makePointsCommand = (useWorldSpaceSize: boolean) => {
     uniform bool useWorldSpaceSize;
     uniform float viewportWidth;
     uniform float viewportHeight;
+    uniform float minPointSize;
+    uniform float maxPointSize;
```

Comment:
I personally wasn't a fan of having to pass this through like this. I feel we should have direct access to the `ALIASED_POINT_SIZE_RANGE` parameter from this environment. But some searching didn't result in anything to retrieve `ALIASED_POINT_SIZE_RANGE` without the `getParameter` on a webgl context. So I gave up on that, if anyone knows how to access the limits without having to pass them in explicitly like this let me know.

Reply:
Unfortunately, that's the only way to do it. We cannot access those values directly from shaders unless we use uniforms. So, you're approach is the correct one.

Reply:
That surprises me, maybe something to do with the shader being indepedent of the environment it's being run in... Thanks for confirming this approach :+1: